lib/testing_appserver/modules/people/user_creation_form.rb

```diff
@@ -21,7 +21,7 @@ class UserCreationForm
     div(:generate_password, xpath: "(//div[@class = 'password-field-wrapper']/div)[2]")
     text_field(:password, xpath: "//input[@name='password']")
 
-    button(:save_form, xpath: "//button[@class='sc-kNPwMy iHxaXx']") # add_id
+    button(:save_form, xpath: "//button[@tabindex='10']") # add_id
```

Comment:
Maybe some other xpath if possible? I'm sure this will break if there some fields will be added

But if you cannot figure out anything - leave it
Maybe ven use eng language in XPath to find it, better than index

Reply:
It's complicated and not possible to find element it with text
<img width="886" alt="Screen Shot 2021-11-23 at 22 58 30" src="https://user-images.githubusercontent.com/40513035/143190050-2fd2af99-3dca-42d3-b9d9-5f29cded6e95.png">